chia/wallet/trading/offer.py

```diff
@@ -198,6 +198,11 @@ def get_requested_amounts(self) -> Dict[Optional[bytes32], int]:
         return requested_amounts
 
     def arbitrage(self) -> Dict[Optional[bytes32], int]:
+        """
+        Returns a dictionary of the amount and type of each asset that is involved in the trade
+        If we are requesting an asset of Type A, then the amount of that asset will be negative
+        However, if we are offering an asset of Type B, then the amount of that asset (int) will be positive.
+        """
```

Comment:
This comment is not correct.  This method has no idea who the party running it is so "we" cannot be a concept.  This method returns a dictionary of assets and how much their offered amount within the offer exceeds/falls short of their requested amount.

Reply:
Yeah im bad at wording things, let me fix it.